README.md

```diff
@@ -403,6 +413,7 @@ Python | 3.6 or higher | No | *N/A* | *Pre-installed or Installed by user* | [PS
 [Intel(R) oneAPI Math Kernel Library](https://software.intel.com/en-us/oneapi/onemkl) | latest | Yes | apt | /opt/intel/inteloneapi/mkl | [Intel Simplified Software License](https://software.intel.com/en-us/license/intel-simplified-software-license)
 [NVIDIA CUDA SDK](https://developer.nvidia.com/cublas) | 10.2 | No | *N/A* | *Installed by user* |[End User License Agreement](https://docs.nvidia.com/cuda/eula/index.html)
 [AMD rocBLAS](https://rocblas.readthedocs.io/en/rocm-4.5.2/) | 4.5 | No | *N/A* | *Installed by user* |[AMD License](https://github.com/ROCmSoftwarePlatform/rocBLAS/blob/develop/LICENSE.md)
+[AMD rocRAND](https://github.com/ROCmSoftwarePlatform/rocRAND) | 5.1.0 | No | *N/A* | *Installed by user* |[AMD License](https://github.com/ROCmSoftwarePlatform/rocRAND/blob/develop/LICENSE.txt)
```

Comment:
We need to use rocRAND 5.1 since previous versions failed one of the oneMKL RNG tests testing skipping ahead some number of random values.

This is the bug report that have been filed, and since fixed: https://github.com/ROCmSoftwarePlatform/rocRAND/issues/234

Reply:
It only has been included in the changelog for 5.1.0 unfortunately: https://github.com/ROCmSoftwarePlatform/rocRAND/blob/develop/CHANGELOG.md#added-1

Reply:
Thanks for the information. We will be able to test with 4.5.2 at this point, and verify the fix when we update our version.